doc/source/whatsnew/v0.19.0.txt

```diff
@@ -1563,3 +1563,4 @@ Bug Fixes
 - ``PeriodIndex`` can now accept ``list`` and ``array`` which contains ``pd.NaT`` (:issue:`13430`)
 - Bug in ``df.groupby`` where ``.median()`` returns arbitrary values if grouped dataframe contains empty bins (:issue:`13629`)
 - Bug in ``Index.copy()`` where ``name`` parameter was ignored (:issue:`14302`)
```

Comment:
moves to 0.20.0

give a user friendly function here, ``_downsample`` is internal.